source/patterns/@aws-solutions-constructs/core/lib/lambda-helper.ts

```diff
@@ -210,4 +210,12 @@ function GetNextId(children: IConstruct[], coreName: string): string {
   });
 
   return `${coreName}-${lastSuffix + 1}`;
+}
+
+export function getLambdaVpcSecurityGroupIds(lambdaFunction: lambda.Function): string[] {
```

Comment:
I didn't notice a unit test for this, did I miss it?

Reply:
Nope, there was no unit test before. Added it in the new code.